internal/http/client.go

```diff
@@ -61,11 +61,11 @@ func NewClient(cfg config.Config) Client {
 	}
 
 	if cfg.BaseURL == "" {
-		cfg.BaseURL = region.DefaultBaseURLs[region.Parse(cfg.Region)]
+		cfg.BaseURL = region.DefaultBaseURLs[region.Parse(string(cfg.Region))]
```

Comment:
It would be neat if we didn't have to convert to strings here if region is a `Region` object.  We should be able to use them as keys in the map.  We can always come back to that later too, not trying to block.